missas/core/facades/google_maps.py

```
@@ -0,0 +1,62 @@
+                "address": formatted_address,
+                "name": place_name,
+                "url": f"https://www.google.com/maps/place/?q=place_id:{result['place_id']}",
+                "full_response": data,
+            }
+
+        logger.info(
+            f"No address found for query: {search_query} (status: {data.get('status')})"
+        )
+        return None
+
+    except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)